apps/src/templates/projects/projectsModule.js

```diff
@@ -1,10 +1,16 @@
 /** @file Redux actions and reducer for the Projects Gallery */
 import {registerReducers} from '@cdo/apps/redux';
+import _ from 'lodash';
 
 // Action types
+
 const TOGGLE_GALLERY = 'projectsModule/TOGGLE_GALLERY';
+const ADD_OLDER_PROJECTS = 'projectsModule/ADD_OLDER_PROJECTS';
+const SET_PROJECT_LISTS = 'projectsModule/SET_PROJECT_LISTS';
+const SET_HAS_OLDER_PROJECTS = 'projectsModule/SET_HAS_OLDER_PROJECTS';
```

Comment:
I'm not really seeing an examples of what you're describing in our codebase, and since `registerReducers` is a thing we wrote, I'm not sure how best to find an external example. Maybe it would be best to have some agreement across the team on how redux actions/reducers should be laid out before spending time moving between the various styles? This particular file seems to be following the redux pattern used in gamelab relatively closely.